app/javascript/mastodon/features/account/components/domain_pill.tsx

```diff
@@ -48,6 +51,8 @@ export const DomainPill: React.FC<{
         {({ props }) => (
           <div
             {...props}
+            role='tooltip'
```

Comment:
I am not sure the `tooltip` role is fitting here. The description of the tooltip ARIA role is:
> A tooltip is a contextual text bubble that displays a description for an element that appears on pointer hover or keyboard focus.

In this case, this is not triggered by hovering, or by focusing the described item.

It is also described as not being in scope for `aria-expanded`.